src/components/containers/PatientsList/__tests__/PatientsList.test.js

```diff
@@ -55,7 +96,49 @@ const allPatients = [
     nhsNumber: '9999999006',
     pasNo: '595941',
     phone: '070 6691 5178',
+    diagnosesDate: '1970-01-01T00:00:00-05:00',
+    ordersDate: '1970-01-01T00:00:00-05:00',
+    resultsDate: '1970-01-01T00:00:00-05:00',
+    vitalsDate: '1970-01-01T00:00:00-05:00',
   },
+];
+const currentPagePatients = [
+  {
+    address: 'Ap #804-4028 Phasellus Rd., Ely, Cambridgeshire, GU67 5CN',
+    dateOfBirth: 128304000000,
+    dateOfBirthConvert: '25-Jan-1974',
+    department: 'Mental Health',
+    diagnosesDateConvert: '17-Jul-2018',
+    gender: 'Female',
+    gpAddress: 'Port Glasgow Practice, 4872 Cubilia St., Port Glasgow, Renfrewshire, TQ3J 4JG',
+    gpName: 'Hawkins Sonya T.',
+    id: '9999999050',
+    name: 'Avram Miller',
+    nhsNumber: '9999999050',
+    ordersDateConvert: '17-Jul-2018',
+    pasNo: '343941',
+    phone: '0845 46 47',
+    resultsDateConvert: '17-Jul-2018',
+    viewPatientNavigation: {},
+    vitalsDateConvert: '17-Jul-2018',
+  },
+  // {
+  //   id: '9999999006',
+  //   address: 'P.O. Box 711, 8725 Purus Rd., Grangemouth, Stirlingshire, B4 8MW',
+  //   dateOfBirth: -363657600000,
+  //   department: 'Primary Care',
+  //   gender: 'Male',
+  //   gpAddress: 'Aulderan Practice, Ap #806-2884 Enim St., Auldearn, Nairnshire, V4F 9GJ',
+  //   gpName: 'Joseph May N.',
+  //   name: 'Ezra Gordon',
+  //   nhsNumber: '9999999006',
+  //   pasNo: '595941',
+  //   phone: '070 6691 5178',
+  //   diagnosesDate: '1970-01-01T00:00:00-05:00',
+  //   ordersDate: '1970-01-01T00:00:00-05:00',
+  //   resultsDate: '1970-01-01T00:00:00-05:00',
+  //   vitalsDate: '1970-01-01T00:00:00-05:00',
+  // },
```

Comment:
Please rewrite parts of the code or delete them. If there is a purpose in commenting out a block of code, we need to write a human-readable text, so anyone can easily understand why the code is commented out